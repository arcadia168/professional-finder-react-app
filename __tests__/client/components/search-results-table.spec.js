import React from 'react';
import SearchResultsTable from '../../../src/client/components/search-results-table';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';
import path from 'path';
import fs from 'fs';

const mockLocalProfessionalsPathName = path.resolve(__dirname, `../../__mocks__/local-professionals-mock.json`);
const mockLocalProfessionals = JSON.parse(fs.readFileSync(mockLocalProfessionalsPathName, 'utf8'));

// const pathName = path.resolve(__dirname, `../../__mocks__/validCardResults.json`);
// const mockCards = fs.readFileSync(pathName, 'utf8');

describe('Search Results', () => {
    const render = customProps => {
        const props = {
            // Default props
            searchResults: mockLocalProfessionals.response.pros,
            ...customProps,
        }
        return mount(<SearchResultsTable {...props} />);
    }
});
