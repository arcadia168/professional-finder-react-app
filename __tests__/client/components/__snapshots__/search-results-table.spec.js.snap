// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Form renders the app as expected 1`] = `
<div
  data-testid="search-form-results__container"
>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
  <h1>
    result
  </h1>
</div>
`;
